src/core/gui/XournalView.cpp

```diff
@@ -712,7 +710,8 @@ void XournalView::layoutPages() {
     Layout* layout = gtk_xournal_get_layout(this->widget);
     layout->recalculate();
 
-    // Todo (fabian): the following lines are conceptually wrong, the Layout::layoutPages function is meant to be called
+    // Todo (fabian): the following lines are conceptually wrong, the Layout::layoutPages function is meant to be
+    // called
     //                by an expose event, but removing it, will break "add page".
```

Comment:
```suggestion
    // called by an expose event, but removing it, will break "add page".
```

Reply:
I found an issue with CTRL+<number> changing the color of the text in the text editor. I will have to look at it tomorrow.